### DISCUSSION TITLE
Support of Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by q-wertz** [lldap/lldap#1031] [General]
Hello,

would it be possible to also publish a pre-build package for Ubuntu 24.04. As far as I can tell, there is only a version for 22.04 and 23.04. 
Or is there a technical reason not to do so?

Best regards.

**Comment by nitnelave** [comment]
@Masgalor WDYT?

**Reply by Masgalor** [reply]
I usually add new build-targets when a new version is published (or upon request).
This has no technical reason, it is to reduce my workload.

That said, the next release `0.6` will have a build for `xUbuntu-24.04`.
(In fact it was already built and will be published later that day, after some testing.)

**Reply by q-wertz** [reply]
Thanks a lot :blush:

**Reply by Masgalor** [reply]
Packages for `0.6.0` are now available. :tada: 
The following build-targets were added: `Leap 15.6, Fedora 40, Fedora 41, xUbuntu 24.04`.
